llvm/test/Analysis/ScalarEvolution/2007-08-06-Unsigned.ll

```diff
@@ -1,32 +1,45 @@
-; RUN: opt < %s "-passes=print<scalar-evolution>" -disable-output 2>&1 | FileCheck %s
+; NOTE: Assertions have been autogenerated by utils/update_analyze_test_checks.py UTC_ARGS: --version 5
+; RUN: opt < %s -passes="print<scalar-evolution>" -disable-output 2>&1 | FileCheck %s
```

Comment:
You can use `-scalar-evolution-classify-expressions=0` to print only the backedge count.